### PR TITLE
Integrate libmediapipe into Akihabara

### PR DESCRIPTION
**Pull request by Speykious**
This pull request is an attempt to integrate `libmediapipe` into Akihabara.

For now, it fixes `MediapipeException` typos (some places were still using `MediaPipeException`) and turns `NativePacketCallback` into a delegate so that it can be used in the definition of the `mp_CalculatorGraph__ObserveOutputStream__PKc_PF` call.

Thus, Akihabara now builds successfully.

However, the method for building and linking `libmediapipe` alongside Akihabara is still unclear. It will eventually be done in this pull request.

### CODE REVIEW

File: src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs [hunk 1]
```diff
@@ -32,7 +34,7 @@ public static extern MpReturnCode
         [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode mp_CalculatorGraph__ObserveOutputStream__PKc_PF(IntPtr graph,
             string streamName,
-            [MarshalAs(UnmanagedType.FunctionPtr)] CalculatorGraph.NativePacketCallback packetCallback,
+            [MarshalAs(UnmanagedType.FunctionPtr)] NativePacketCallback packetCallback,
```
Comment (sr229) [inline src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: How was this sourced? if you referenced on the Unity repo, there's a calculator graph class generated from a protofile. We shouldn't touch it unless you know what it is.

**Review by sr229 — CHANGES_REQUESTED** [review]

File: src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs (hunk 1 above)
Comment (Speykious) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: Should we just integrate this class from the Unity repo into `ProtoCalculator`?

Comment (sr229) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: Did you check `build.py`? It's getting some protofiles and protocs them, so figure that one out first.

Comment (Transfusion) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: This was sourced here, 
https://github.com/homuler/MediaPipeUnityPlugin/blob/92dd55579f2faeabf49338974ddb1d15ae54b683/Packages/com.github.homuler.mediapipe/Runtime/Scripts/Framework/CalculatorGraph.cs#L8

MediaPipeUnity's "`CalculatorGraph.cs`" is just a shim for `CalculatorGraph_Unsafe.cs`

Comment (Speykious) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: What's `CalculatorGraph_Unsafe.cs`?

Comment (sr229) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: We're looking at this a bit incorrectly, you can refer here: https://github.com/homuler/MediaPipeUnityPlugin/issues/235

Comment (Speykious) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: Does this mean that we already have what it takes? It seems like it's defined in the C#-generated protobuf code from the previous pull request:
image

Comment (sr229) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: Looks like it. Is this ready to be merged if that's the only concern?

Comment (Speykious) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: Well, if we undo the modification above, Akihabara simply won't compile. From where then are we supposed to take the `NativePacketCallback` type?

Comment (sr229) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: > Well, if we undo the modification above, Akihabara simply won't compile. From where then are we supposed to take the `NativePacketCallback` type?

Check from the C code to see if that's correct type for the delegate, if it is, then I can approve it.

**Review by sr229 — APPROVED** [review]
I'll approve this ahead of time but check the type for `NativePacketCallback`  if it's correct then I'll start to merge.

File: src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs (hunk 1 above)
Comment (Speykious) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: This is the code from `mediapipe_api/framework/calculator_graph.cc` (l.69):
```cpp
MP_CAPI(MpReturnCode) mp_CalculatorGraph__ObserveOutputStream__PKc_PF(mediapipe::CalculatorGraph* graph,
                                                                      const char* stream_name,
                                                                      NativePacketCallback* packet_callback,
                                                                      absl::Status** status_out) {
  TRY {
    auto status = graph->ObserveOutputStream(stream_name, [packet_callback](const mediapipe::Packet& packet) -> ::absl::Status {
      return absl::Status { std::move(*packet_callback(packet)) };
    });
    *status_out = new absl::Status { std::move(status) };
    RETURN_CODE(MpReturnCode::Success);
  } CATCH_EXCEPTION
}
```

The `NativePacketCallback` type is defined under `mediapipe_api/framework/calculator_graph.h` (l.23) as the following function:
```cpp
typedef absl::Status* NativePacketCallback(const mediapipe::Packet&);
```

Comment (sr229) [reply src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs:37]: That returns a Integer so `IntPtr` is correct. Thanks for investigating.

**Comment by Speykious** [comment]
Alright, thanks for merging! Unit tests will come in a separate PR then

**Pull request merged**